### PR TITLE
Upgrade Acquia Cloud Api endpoints

### PR DESCRIPTION
**Pull request by adeelkhan**
PROD-1735

### CODE REVIEW

File: tubular/drupal.py [hunk 1]
```diff
@@ -109,25 +116,91 @@ def parse_response(response, error_message):
...
+            secret (str): The Acquia api secret key to run the command.
+            path_name (str): The path to write the tag name to.
+
+        Returns:
+            environment id (str): The identifier is a key consisting of the internal database ID
+            of the environment and the application UUID
+
+        Raises:
+            KeyError: Raised if env value is invalid.
+        """
```
Comment (adzuci) [inline tubular/drupal.py]: Nit: You may want to unindent this.

File: tubular/drupal.py [hunk 2]
```diff
@@ -109,25 +116,91 @@ def parse_response(response, error_message):
...
+
+
+def fetch_environment_uid(app_id, env, token):
+    """
+        Fetches environment uid based on environment name
+
+        Args:
+            app_id (str): Application id assigned to Drupal instance.
+            env (str): The environment to clear varnish caches in (e.g. test or prod)
+            client_id (str): The Acquia api client id necessary to run the command.
```
Comment (adzuci) [inline tubular/drupal.py]: You may want to check docstrings before you merge this, son args may need to be updated.

**Review by adzuci — COMMENTED** [review]
Most of this looks good to me, if you thought been may need to roll back you could split this PR into two to use an expand and contract style roll out and include support for both APIs for a period big time, but that may be overkill.

**Review by Dillon-Dumesnil — APPROVED** [review]
I only scanned it, but nothing looks wrong to me. I'd wait to do any merging until someone from the website squad has reviewed as well.

**Review by reillz10 — APPROVED** [review]
👍  looks good to me!

**Pull request merged**